pkg/podvolume/util.go

```diff
@@ -295,3 +297,17 @@ func GetPodVolumesUsingRestic(pod *corev1api.Pod, defaultVolumesToRestic bool) [
 	}
 	return podVolumes
 }
+
+// isPodQualified checks if the pod's status is qualified for a PVB/PVR to backup/restore its volumes.
+// If no, return the error found
+func isPodQualified(pod *corev1api.Pod) error {
+	if pod.Spec.NodeName == "" {
+		return errors.Errorf("pod is not scheduled, name=%s, namespace=%s, phase=%s", pod.Name, pod.Namespace, pod.Status.Phase)
+	}
+
+	if pod.Status.Phase != corev1api.PodRunning {
+		return errors.Errorf("pod is not running, name=%s, namespace=%s, phase=%s", pod.Name, pod.Namespace, pod.Status.Phase)
+	}
```

Comment:
You also need to check deletion timestamp. A "Terminating" pod will still have a 'Running" phase while the pod cleans up prior to exit. 
 `pod.Status.Phase != corev1api.PodRunning || pod.DeletionTimestamp != nil`

Reply:
Done, added the check of `pod.DeletionTimestamp != nil`. Thanks.